spring-cloud-services-service-registry-autoconfigure/build.gradle

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2002-2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+description = "Spring Cloud Services Service Registry Autoconfigure"
+
+dependencies {
+    implementation("org.springframework.boot:spring-boot-starter")
```

Comment:
`api` is probably better here

Reply:
@royclarkson I didn't think we would want any transitive dependencies coming from the autoconfig modules. Is this not the usual convention? I wasn't sure what to use a template tbh.

Reply:
It's better to think in terms of the POM being produced. Dependencies generated from Gradle will either be `compile` or `runtime` scope in Maven POMs. Unless there are specific reasons not to, we should produce `compile` scoped dependencies that an implementing application can utilize.

Reply:
ok makes sense! My bad too as I was thinking of `implementation` as not transitive when you're much more correct in saying that it's `runtime`. It's still transitive either way.